.github/workflows/wallet-with-console-smoke.yml

```diff
@@ -0,0 +1,53 @@
+name: Smoke Test - Wallet with Console
+
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+
+jobs:
+  console-smoke-test:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Setup Node.js Latest
+        uses: actions/setup-node@v3
+        with:
+          node-version: '*'
+          registry-url: 'https://registry.npmjs.org'
+
+      - name: Install chrome and chromedriver
+        run: |
+          bash .github/scripts/install_chromedriver.sh
+          bash .github/scripts/install_chrome.sh
+
+      - name: Install dependencies
+        run: yarn install
+
+      - name: Build
+        run: yarn build:test-console
```

Comment:
endpoint is different and I did not want to be wedded to it in case we canted to change configuration for test purposes, but I guess I could potentially import the fairground config and override the endpoint for now whilst they are similar

Reply:
also we append a custom key so we get a stable endpoint for testing. Did not want to bind that logic to the fairground config

Reply:
Yeah we import + override in the dev config. I think that would be a good shout in this too